libs/execution_context/ebpf_maps.c

```diff
@@ -57,6 +66,7 @@ typedef struct _ebpf_map_function_table
         _In_ ebpf_core_map_t* map, _In_ const uint8_t* key, _In_ const uint8_t* value, ebpf_map_option_t option);
     ebpf_result_t (*delete_entry)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* key);
     ebpf_result_t (*next_key)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* previous_key, _Out_ uint8_t* next_key);
+    uint8_t* (*find_and_delete_entry)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* key);
```

Comment:
since line 59 now has "delete_on_success", why do you need a separate entry for this?